src/Angor/Client/Pages/Create.razor

```diff
@@ -222,26 +306,45 @@
             }
         },
         Metadata = new(),
-        CreationTransactionId = "unknowen",
+        CreationTransactionId = null,
     };
-    
-    bool nostrProfileCreated;
-    bool applicationDataOnNostr;
+
+    private int activeTab = 1;
+
+    bool createProfileSpinner = false;
+    bool createApplicationDataSpinner = false;
+    bool creatProjectSpinner = false;
+    bool publishProjectSpinner = false;
 
     protected override async Task OnInitializedAsync()
     {
         if (hasWallet)
         {
             var projects = storage.GetFounderProjects();
             var keys = _walletStorage.GetFounderKeys();
-            var latestProject = projects.MaxBy(p => p.ProjectIndex);
+
+            if (projects.Where(_ => !string.IsNullOrEmpty(_.CreationTransactionId)).ToList().Count == keys.Keys.Count)
+            {
+                throw new ApplicationException("all founder keys have been used for this wallet");
```

Comment:
This should not be an exception, it's valid unless you want to block the option in the founder page

Reply:
ah so just give a message to the user then instead of an exception?